src/App.tsx

```diff
@@ -1,61 +1,124 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+export const App: React.FC = () => {
+  const [titleInput, setTitleInput] = useState('');
+  const [userSelect, setUserSelect] = useState(0);
+  const [titleFull, setTitleFull] = useState(true);
+  const [userChosen, setUserChosen] = useState(true);
+  const [todoId, setTodoId] = useState(
+    Math.max(...todosFromServer.map(todo => todo.id)),
+  );
+
+  const toDosUser = todosFromServer.map((todo) => {
+    return {
+      ...todo,
+      user: usersFromServer
+        .find((userFromId): boolean => userFromId.id === todo.userId),
+    };
+  });
+
+  class AddedTodos {
```

Comment:
why did you create class here? 😲
you can use object for new todo. If you use Class, should create it in new file